<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MVCDemoClass</class>
 <widget class="QWidget" name="MVCDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MVCDemo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <widget class="MyListView" name="listView1"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
       <item>
        <widget class="QQuickWidget" name="quickWidget">
         <property name="resizeMode">
          <enum>QQuickWidget::SizeRootObjectToView</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>MyListView</class>
   <extends>QListView</extends>
   <header>MyListMVC.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MVCDemo.qrc"/>
 </resources>
 <connections/>
</ui>
